packages/kit/src/runtime/load.js

```diff
@@ -3,6 +3,10 @@
  * @returns {import('types').NormalizedLoadOutput}
  */
 export function normalize(loaded) {
+	if (!loaded) {
+		return {};
+	}
+
```

Comment:
Does this mean that returning a falsy value from a `load` function is now equivalent to returning `{}`? Is this desirable? If there were other corresponding changes to public types for this, I missed them.

Reply:
Oh, I see elsewhere you also remove a comment `// TODO do we still want to enforce this now that there's no fallthrough?`, so, yes, it sounds like this is a conscious decision made at the same time as fixing the main bug. The types should still be updated.

Reply:
updated the types and the docs